etc/lfcd.fish

```diff
@@ -11,17 +11,8 @@
 #
 # You may put this in a function called fish_user_key_bindings.
 
-function lfcd
-    set tmp (mktemp)
-    # `command` is needed in case `lfcd` is aliased to `lf`
-    command lf -last-dir-path=$tmp $argv
-    if test -f "$tmp"
-        set dir (cat $tmp)
-        rm -f $tmp
-        if test -d "$dir"
-            if test "$dir" != (pwd)
-                cd $dir
-            end
-        end
-    end
+function lfcd --wraps="lf" --description="lf - Terminal file manager (changing directory on exit)"
+    # `command` is needed in case `lfcd` is aliased to `lf`,
```

Comment:
Minor nit: Instead of `, and quotes`, I'd start another sentence and write `. Quotes`.

Reply:
Indeed, looks better with a new sentence. I also now think the wording "cause `cd` to fail" is a bit vague (without context it might not be clear why exactly this is desired) so expanded on this bit too.

Reply:
Thanks! I do now have an even more minor nit: if we focus on not changing the directory, I'd say:

> Quotes will prevent `cd` from changing the directory if `lf` prints nothing to stdout due to an error.

I also think that I mildly prefer talking about `cd` failing, since the user will probably see the error message in that case.

But it's pretty much OK as is, as well.